package.json

```diff
@@ -6,14 +6,17 @@
     "clean": "bundle exec jekyll clean",
     "reset": "npx rimraf .git",
     "start": "bundle exec jekyll serve",
-    "test": "bundle exec htmlproofer _site; npx a11y '_site/**/*.html'"
+    "start-detached": "bundle exec jekyll serve --detach",
+    "test": "bundle exec htmlproofer _site; npx a11y '_site/**/*.html'",
```

Comment:
If you're replacing `a11y`, there are some references here that you'll need to remove also.

Once you remove the offending `a11y` package and this extraneous ` npx a11y '_site/**/*.html'` from `test` here, you may uncover a hidden pre-existing problem with `htmlproofer` where it's producing false-negative test results due to differences in `site.url` vs `localhost` internal URLs.

At least, that's what happening on my downstream site.  I'm still troubleshooting, but interested to see what you do here.

Reply:
I went ahead and removed `a11y` and rebased this PR. I ran the `npm test` locally and it seemed to pass for me.

Reply:
I think the problems start when you set a `site.url` in `_config.yml`.

Reply:
Right and for another Jekyll site (https://github.com/CivicActions/accessibility/blob/main/package.json#L13) we replace the live site URL with the internal one for pa11y-ci.

htmlproofer does have a `url_swap` option that might be able to do the same thing.

Reply:
The one thing I realized is that package.json won't be able to get the site URL so you have to manage your own versions with the specific site URLs.

Reply:
For `htmlproofer`, I was able troubleshoot this a bit more.  This is caused by the `meta` absolute reference in `<link rel="canonical" href="{{ page.url | absolute_url }}" />` that is produced in the `build`.  There's two solutions:

1) use a custom swap in the `htmlproofer` command line, to basically remove `site.url` name: `--url-swap "https\://digitalcorps.gsa.gov:"`.

2) add an ignore statement in the `canonical`:

```html
 <link rel="canonical" href="{{ page.url | absolute_url }}" data-proofer-ignore />
```